### DISCUSSION TITLE
How to do annotation for code block inside content tabs

### DISCUSSION THREAD

**Discussion by binooetomo** [squidfunk/mkdocs-material#7111] [Q&A] [answered]
Dear all.

My md file have 2 code blocks each with it's own annotation.
<code>
``` { .toml .annotate title="TAB-A" }

Line 1 # (1)
```

1.  This is Line 1 of TAB-A
</code>
<code>

``` { .toml .annotate title="TAB-B" }

Line 1 # (1)
```

1.  This is Line 1 of TAB-B

</code>

It work like what I expected.

Now I want to 'group' that 2 code block inside a content tabs.
This is my try

<code>
=== TAB-A

    ``` { .toml .annotate title="TAB-A" }

    Line 1 # (1)
    ```

=== TAB-B

    ``` { .toml .annotate title="TAB-B" }

    Line 1 # (2)
    ```

1.  This is Line 1 of TAB-A
2.  This is Line 1 of TAB-B

</code>

But that one didn't work as expected.
So, How to do code annotation inside content tab?

Sincerely   
-bino-

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @binooetomo,
you can see annotations used in content tabs on the "Publishing your site" docs page:
- https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions-material-for-mkdocs
- https://github.com/squidfunk/mkdocs-material/blob/9d901a3caf25f6222cded36526cdf0bdd471fde8/docs/publishing-your-site.md?plain=1#L56

Comparing it to your example, we can see that numeric list should be restarted on every content tab instead of defining a global list under all tabs.
